website/docs/components/code-block/partials/guidelines/guidelines.md

```diff
@@ -69,6 +69,22 @@ Line numbers are displayed by default and can make longer blocks of code and sni
 In the Figma component, the code examples have the appropriate number of lines by default but must be manually hidden or shown to match the length of custom snippets.
 !!!
 
+## Height toggle button
+
+Use the height toggle button to let users expand and collapse longer code snippets.
+
+When `maxHeight` is set and the code content is taller than that value, a **Show more code** button appears at the bottom of the Code Block.
+
+![Collapsed Code Block showing limited lines and a 'Show more code' button at the bottom.](/assets/components/code-block/code-block-collapsed.png)
+
+- Clicking this button removes the height limit, allowing the full code snippet to be displayed. 
```

Comment:
I wouldn't bullet these out since they are single items.

```suggestion
Interacting with this button removes the height limit, expanding the Code Block to display the full code snippet. 
```